src/adapter/src/catalog/builtin_table_updates.rs

```diff
@@ -277,6 +277,13 @@ impl CatalogState {
             }
         };
 
+        if !entry.item().is_temporary() {
+            // Clean up the `mz_object_dependencies` table.
```

Comment:
"Clean up" is maybe not the most accurate statement because we could be adding something as well.

Reply:
oh yeah!